tests/providers/amazon/aws/operators/test_redshift_cluster.py

```diff
@@ -127,3 +128,43 @@ def test_pause_cluster_not_called_when_cluster_is_not_available(self, mock_get_c
         )
         redshift_operator.execute(None)
         mock_get_conn.return_value.pause_cluster.assert_not_called()
+
+
+class TestDeleteClusterOperator:
+    def test_init(self):
+        redshift_operator = RedshiftDeleteClusterOperator(
+            task_id="task_test", cluster_identifier="test_cluster", aws_conn_id="aws_conn_test"
+        )
+        assert redshift_operator.task_id == "task_test"
+        assert redshift_operator.cluster_identifier == "test_cluster"
```

Comment:
We don't need this test, doesn't add much value

Reply:
Thanks for cleaning it.